### PR TITLE
Fixed incorectly displayed min replicas in kubectl get hpa

### PR DESCRIPTION
**Pull request by piosz**
cc @fgrzadkowski @jszczepkowski

### CODE REVIEW

**Comment by wojtek-t** [comment]
LGTM

**Comment by piosz** [comment]
Shippable is flaky

File: pkg/kubectl/resource_printer.go [hunk 1]
```diff
@@ -1396,7 +1396,7 @@ func printHorizontalPodAutoscaler(hpa *extensions.HorizontalPodAutoscaler, w io.
 	if hpa.Status.CurrentCPUUtilizationPercentage != nil {
 		current = fmt.Sprintf("%d%%", *hpa.Status.CurrentCPUUtilizationPercentage)
 	}
-	minPods := hpa.Spec.MinReplicas
+	minPods := *hpa.Spec.MinReplicas
```
Comment (jszczepkowski) [inline pkg/kubectl/resource_printer.go]: Shouldn't you handle nil case here?

Comment (piosz) [inline pkg/kubectl/resource_printer.go]: Won't it be set to the default value? I'll verify it tomorrow.

Comment (jszczepkowski) [inline pkg/kubectl/resource_printer.go]: It will be set. However, I'm not sure whether we should depend on it.

**Comment by hw-qiaolei** [comment]
It seems this PR is duplicated with #15902

**Comment by piosz** [comment]
closing in favor of #15902

**Pull request closed without merging**